src/process.rs

```diff
@@ -115,3 +122,49 @@ pub fn init(ctx: &Ctx<'_>) -> Result<()> {
 
     Ok(())
 }
+
+pub struct ProcessModule;
+
+impl ModuleDef for ProcessModule {
+    fn declare(declare: &mut Declarations) -> Result<()> {
+        declare.declare("env")?;
+        declare.declare("cwd")?;
+        declare.declare("argv0")?;
+        declare.declare("id")?;
+        declare.declare("argv")?;
+        declare.declare("platform")?;
+        declare.declare("arch")?;
+        declare.declare("hrtime")?;
+        declare.declare("release")?;
+        declare.declare("version")?;
+        declare.declare("versions")?;
+        declare.declare("exit")?;
+
+        declare.declare("default")?;
+        Ok(())
+    }
+
+    fn evaluate<'js>(ctx: &Ctx<'js>, exports: &mut Exports<'js>) -> Result<()> {
+        let globals = ctx.globals();
+        let process: Object = globals.get("process")?;
+
+        export_default(ctx, exports, |default| {
+            default.set("env", process.get::<&str, Value>("env"))?;
```

Comment:
We can iterate through the enumarable keys of the process object and then set them accordingly. That way we can get rid of the repeat:
 export_default(ctx, exports, |default| {
 
      for key in process.keys() {
         ...
      }
 }

Reply:
I wanted to do similar like it is done in `timers`.
However `timers` has all props as `Function` type
```
let function: Function = globals.get(func_name)?;
```
But for `process` it didn't work, bc all props are different types.
could you give me some snippet how to make it generic?

Reply:
I found better solution